symfony/templates/workqueue/consents.html.twig

```diff
@@ -41,11 +41,14 @@
             {% include 'partials/workqueue-consents-advance-filters.html.twig' %}
         </div>
         <div class="col-md-9">
-            {% if canExport %}
-                <p class="text-right">
+            <div class="text-right">
+                <div class="page-drop-down">
+                    Page <select name="pageDropDown" class="input-sm"></select>&nbsp;of&nbsp;<span class="total-pages"></span>
```

Comment:
This select doesn't match the style of all our other dropdowns. I'm guessing you might have not added the `form-control` class because it makes the select a block element. But if you add the `form-inline` class to one of the parent divs, it works pretty well.

So, something like:

```suggestion
            <div class="text-right form-inline">
                <div class="page-drop-down">
                    Page <select name="pageDropDown" class="form-control input-sm"></select>&nbsp;of&nbsp;<span class="total-pages"></span>
```

And since this is repeated four times (twice on each work queue view), can you move this whole block to a separate partial template?